src/common/store/cache/tracks/sagas.js

```diff
@@ -410,11 +413,19 @@ function* watchFetchCoverArt() {
 
       const rgb = yield call(averageRgb, url)
       yield put(
-        setColor({
+        setAverageColor({
           multihash,
           color: rgb
         })
       )
+
+      const dominantColors = yield call(dominantColor, url)
+      yield put(
+        setDominantColors({
+          multihash,
+          colors: dominantColors
+        })
+      )
```

Comment:
This saga runs on every track fetch which is kinda unfortunate since most of the time you won't be playing the track.

Do you think we could fetch these dominant colors and call `setDominantColors` somewhere from the visualizer itself? Or perhaps when the track is played?

Alternatively, let's just use dominant color for the CurrentlyPlaying, NowPlaying, etc. which I think is far better than average color anyway and not call `yield call(averageRgb, url)` here at all.

wdyt?